### ISSUE TITLE
Set-AzNetworkWatcherConfigFlowLog - Cannot bind parameter 'Workspace'

### ISSUE THREAD

**Issue by JorgeDios** [Azure/azure-powershell#12135]
The cmdlet **Set-AzNetworkWatcherConfigFlowLog** fails with the following error when using the parameter **workspace**:

##[error]Cannot bind parameter 'Workspace'. Cannot convert the "Microsoft.Azure.Commands.OperationalInsights.Models.PSWorkspace" value of type "Microsoft.Azure.Commands.OperationalInsights.Models.PSWorkspace" to type "Microsoft.Azure.Management.Internal.Network.Common.IOperationalInsightWorkspace".

The **Example 3: Configure Flow Logging and Traffic Analytics for a Specified NSG** in the documentation shows how to use _workspace_ parameter by passing the value retrieved by the **Get-AzOperationalInsightsWorkspace**. 

```Powershell
$workspace = Get-AzOperationalInsightsWorkspace -Name WorkspaceName -ResourceGroupName WorkspaceRg
Set-AzNetworkWatcherConfigFlowLog -NetworkWatcher $NW -TargetResourceId $nsg.Id -EnableFlowLog $true -StorageAccountId $storageID -EnableTrafficAnalytics -Workspace $workspace -TrafficAnalyticsInterval 60
```

This worked in previous versions, but not anymore since the version 4.1.0 of the Az module.

**Comment by dingmeng-xue** [comment]
@JorgeDios , each Az major release will introduce breaking changes due to service upgrade or deprecated.  We will look into you case and understand the root cause.

@VeryEarly , please check this question.

**Comment by VeryEarly** [comment]
fixed in #12155
